packages/multichain/src/scope/authorization.ts

```diff
@@ -35,8 +35,8 @@ export const validateAndNormalizeScopes = (
   requiredScopes: ExternalScopesObject,
```

Comment:
are we still using `validateAndNormalizeScopes` anywhere? Not seeing it used in this package or either extension side PRs 🤔 If we are lets remember to rename it!

Reply:
https://github.com/MetaMask/metamask-extension/pull/27782/files#diff-595dd2fe0233f8be65cd7bd3295a5e882fd8ebf623be4a4c0055c8d7278b6444R125